time_test.go

```diff
@@ -0,0 +1,45 @@
+package mono
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+)
+
+func TestTime_JsonUnmarshal(t *testing.T) {
```

Comment:
Usually, we call tests with the following name convention: `Test<Type>_<Method>`.
Let's call this test `TestTime_ UnmarshalJSON`.